templates/vimeoTranscript.hbs

```diff
@@ -0,0 +1,32 @@
+<div class="media-transcript-button-container">
+    {{#if _inlineTranscript}}
+        <button class="media-inline-transcript-button button" aria-expanded="false">
+            <div class="transcript-text-container">
+                {{#if inlineTranscriptButton}}
+                    {{inlineTranscriptButton}}
+                {{else}}
```

Comment:
drop the `else` block, can't see why you'd ever want the external transcript url to be shown as the label for the inline transcript button!